lab-doug/app/component/create-list-form/create-list-form.html

```diff
@@ -0,0 +1,7 @@
+<form class="form-inline create-list-form" novalidate ng-submit="createListFormCtrl.createList()">
+  <div class="form-group">
+    <label for="name">First Name:</label>
```

Comment:
The for attribute of a label tag needs to be the id of the element it is labelling, not the name of it.